lib/rubocop/cop/layout/align_hash.rb

```diff
@@ -10,6 +10,7 @@ module Layout
       #   - key (left align keys, one space before hash rockets and values)
       #   - separator (align hash rockets and colons, right align keys)
       #   - table (left align keys, hash rockets, and values)
+      #   - or a list of the above options
```

Comment:
I think this should say something like "Alternatively you can specify multiple allowed styles. That's done by passing a list of styles to `EnforcedStyles`. " (or something along those lines)